x-pack/solutions/search/plugins/enterprise_search/common/types/indices.ts

```diff
@@ -14,6 +14,7 @@ export interface AlwaysShowPattern {
   index_pattern: string;
 }
 
+// TODO: Remove this type
```

Comment:
yes!

Reply:
Will remove this with the server code too.